lib/dust.js

```diff
@@ -270,18 +308,19 @@
     }
   };
 
-  function Context(stack, global, blocks, templateName) {
+  function Context(stack, global, options, blocks, templateName) {
```

Comment:
To pass the minimum number of params for makeBase. But this is completely
internal so it can be whatever if you like.

On Fri, Apr 24, 2015, 6:59 AM Jimmy Chan <notifications@github.com> wrote:

> In lib/dust.js
> <https://github.com/linkedin/dustjs/pull/641#discussion_r29050275>:
>
> > @@ -270,18 +308,19 @@
> >      }
> >    };
> >
> > -  function Context(stack, global, blocks, templateName) {
> > +  function Context(stack, global, options, blocks, templateName) {
>
> nit: why this ordering? IMO global and blocks kinda go together.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/linkedin/dustjs/pull/641/files#r29050275>.
>